maps/torch/items/summarydocuments.dm

```diff
@@ -1,393 +1,375 @@
 
 /obj/item/documents/generic/sol/comm
 	name = "ZA-Common reminder"
-	desc = "A memo that has seen better days. It's been crumpled up, dried in coffee and put back up by some vandals. From what's left, it reads: 'Speaking good ZA-Common is a must to be an excellent head of staff. Please exercise moderation with accents.'"
-
-/obj/item/document/generic/sol/holocallloiter
-	name = "holocall etiquette
-	desc = "\"Internal use ONLY\" reports, requesting people that if a holocall lingers, they should make a point to end it.
+	desc = "A memo that has seen better days. It's been crumpled up, dried in coffee and put back up by some vandals. From what's left, it reads: \"Speaking good ZA-Common is a must to be an excellent head of staff. Please exercise moderation with accents.\""
 
 /obj/item/documents/generic/sol/punishments
 	name = "disposition of offenses"
-	desc = "\"For Official Use Only\" memorandum detailing a commander’s options with regards to punishing uniformed servicemembers. It seems that, in ascending order of severity, a commander can: take no action (after investigation), take administrative action, impose nonjudicial punishment, and court-martial the offender."
+	desc = "A \"for official use only\" memorandum detailing an officer's options with regards to punishing uniformed servicemembers. In ascending order of severity, the officer can: take no action (after investigation), take administrative action, impose nonjudicial punishment, and court-martial the offender."
 
 /obj/item/documents/generic/sol/adminactions
 	name = "administrative actions"
-	desc = "\"For Official Use Only\" memorandum detailing a commander’s options with regards to administrative action, as referenced in the disposition of offenses. Options include: counseling, corrective training, administrative reprimands, bar to reenlistment, suspension of favorable personnel actions (flagging), MOS reclassification, revocation of security clearance, relief from duties, adverse NCOER/OER, removal from special status, rehabilitative transfer, administrative reduction, administrative separations."
+	desc = "A \"for official use only\" memorandum detailing an officer's options with regards to administrative action as referenced in the disposition of offenses. Options include: counseling, corrective training, administrative reprimands, bar to reenlistment, suspension of favorable personnel actions (flagging), MOS reclassification, revocation of security clearance, relief from duties, adverse NCOER/OER, removal from special status, rehabilitative transfer, administrative reduction, and administrative separations."
```

Comment:
/sol/adminactions was highlighted as being unacceptable, due to the obvious US-inspired nature.

If we drop the MOS stuff, and try to generalize, it might pass.